engine/engine/content/builtins/scripts/debugger.lua

```diff
@@ -121,10 +123,10 @@ local function set_lightweight_hook(force, co)
     continue_execution = false
     if co then
       debug.sethook(co, nil, "lcr")
-      debug.sethook(co, lightweight_hook, "c")
+      sys.set_debugger_lightweight_hook(co, lightweight_hook)
     else
       debug.sethook(nil, "lcr")
-      debug.sethook(lightweight_hook, "c")
+      sys.set_debugger_lightweight_hook(lightweight_hook)
```

Comment:
using of the new lightweight hook, which does just a few operations on the cpp side